docs/basic/getting-started/function-components.md

```diff
@@ -11,14 +11,14 @@ type AppProps = {
   message: string;
 }; /* use `interface` if exporting so that consumers can extend */
 
-// Easiest way to declare a Function Component; return type is inferred.
-const App = ({ message }: AppProps) => <div>{message}</div>;
+// Easiest way to declare a Function Component that expects a prop value; return type is inferred.
+const App: React.FC<Props> = ({ message }: AppProps) => <div>{message}</div>;
```

Comment:
in this case the return type is not inferred, is it as stated in the declaration of `React.FC` and it is `ReactElement`.